internal/ovirt/resource_ovirt_vm.go

```diff
@@ -377,7 +382,16 @@ func handleVMPlacementPolicy(
 		}
 		hasPlacementPolicy = true
 	}
+
 	if hasPlacementPolicy {
+		if isSupported, _ := client.SupportsFeature(ovirtclient.FeaturePlacementPolicy); !isSupported {
```

Comment:
We must handle the error here as the call to oVirt can fail. That error must be logged.